lib/runner/cli/cli.js

```diff
@@ -10,6 +10,7 @@ const Runner = require('../runner.js');
 const ProcessListener = require('../process-listener.js');
 const analyticsCollector = require('../../utils/analytics.js');
 const {RealIosDeviceIdError, iosRealDeviceUDID, isRealIos, isMobile, killSimulator, isAndroid} = require('../../utils/mobile.js');
+const {merge, union} = require('lodash');
```

Comment:
lodash isn't a dependency of nightwatch.

Reply:
also there are a few other new dependencies which don't seem to be used.

Reply:
updated